lib/partition_setup.pm

```diff
@@ -38,6 +38,11 @@ our %partition_roles = qw(
   raw alt-a
 );
 
+# We got changes to the storage-ng UI in SLE 15 SP1, Leap 15.1 and TW
+sub is_storage_ng_newui {
+    return is_storage_ng && (is_sle('15-SP1+') || (is_opensuse && !is_leap('<=15')));
```

Comment:
Seems that my old comment got folded up. As we don't have these changes in TW we need to exclude it. I wrote it this way because I was optimistic. Let's change it to
`return is_storage_ng && (is_sle('15-SP1+') || is_leap('15.1+'))`

Reply:
updated

Reply:
Thanks! I will merge once travis is done